src/sinks/tsd-query-log-sink.ts

```diff
@@ -72,29 +74,35 @@ export class TsdQueryLogSink implements tsdDef.Sink {
     private static stringify(metricsEvent:tsdDef.MetricsEvent):string {
         var transformedMetricsEvent = TsdQueryLogSink.transformMetricsEvent(metricsEvent);
 
-        return JSON.stringify(utils.stenofy(transformedMetricsEvent),
+        return JSON.stringify(utils.stenofy(transformedMetricsEvent, transformedMetricsEvent.annotations["_host"]),
```

Comment:
Ah, good one. Did the suppression, which then requires the hostname to be passed in separately as it will not be present in the `transformedMetricsEvent`.